### DISCUSSION TITLE
Datagrid - Detail Pane

### DISCUSSION THREAD

**Discussion by jackieldev** [vmware/clarity#6122] [Q&A] [answered]
Is it possible for a specific size in the column when opening details panel?

https://clarity.design/angular-components/datagrid/#detail-pane

![image](https://user-images.githubusercontent.com/51405114/124515309-867ba000-ddb5-11eb-9c95-67714a34221d.png)

Thanks for help.

**Comment by bbogdanov** [comment, marked as answer]
Hi @jackieldev,

Could you please give us more information about your scenario?

As a design system we have certain guidelines that we have to address in question like yours but we need to know a little bit about the  `why` in order to help you with the `how`.

Thank you in advance!

**Reply by jackieldev** [reply]
Hi @bbogdanov,

Yes. Below is an example. This area in the red square can be used for details.

![image](https://user-images.githubusercontent.com/51405114/124587556-1fe69880-de2e-11eb-813f-242a51843a5f.png)

**Reply by bbogdanov** [reply]
Ok. as I can see `.datagrid-detail-open .datagrid-inner-wrapper` css selector sets a `width="34%"` so what I could suggest is just overwrite it with `width="fit-content"` or `width="auto"`.

Because it is Angular you might need `::ng-deep`

**Reply by jackieldev** [reply]
Thanks @bbogdanov , I forgot this ::ng-deep.
